revert hide flags addition (#617)
bugsnag/bugsnag-unity: CHANGELOG.md

* Added `BugsnagEvent.FeatureFlags` to allow feature flags to be queried before event delivery.
  [#613](https://github.com/bugsnag/bugsnag-unity/pull/613)

### Bug fixes

> * Reverted the hide flags bug fix introduced [here](https://github.com/bugsnag/bugsnag-unity/pull/604) as it was causing null ref errors when using Bugsnag in the editor. Please see this PR for more details

Reviewer: ```suggestion
* Reverted the `HideFlags` fix introduced [here](https://github.com/bugsnag/bugsnag-unity/pull/604) as it was causing (harmless) `NullReferenceException`s in the editor.
```
